service/system/systemd-geopm.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=User space interface to hardware features with fine grained access control
+
+[Service]
+Environment=PYTHONUNBUFFERED=true
```

Comment:
This reminds me, do we need to do anything special for output, like using a systemd logger or something? Or is print-based logging sufficient?

Reply:
Good question, I've certainly seen special logging it in some examples for a variety of reasons.  We can look into it.   Messages are turning up in the syslog, but we may want our own log file.